AUTHORS

```diff
@@ -8,3 +8,4 @@ Vin Ganeshan <ganeshan.vinayak@gmail.com>
 Vishal Kal <vishal.kal0@gmail.com>
 Vishal Kalyanasundaram <vishalkal@berkeley.edu>
 William Chieng <wchieng@berkeley.edu>
+m-j-mcdonald <m_j_mcdonald@comcast.net>
```

Comment:
Change your git config so that your full name appears here instead of the handle